frontends/p4/substitution.h

```diff
@@ -76,21 +76,11 @@ class TypeVariableSubstitution final : public TypeSubstitution<const IR::ITypeVa
  public:
     TypeVariableSubstitution() = default;
     TypeVariableSubstitution(const TypeVariableSubstitution& other) = default;
-    /* Set bindings for multiple variables at the same time.
-     * @param params   List of variables to set.
-     * @param args     List of type arguments to set to variables.
-     * @param root     P4 program element which is being processed. Used for error reporting.
-     * @return         True on success. */
-    bool setBindings(const IR::Node* root,
```

Comment:
As you can tell, this is a JavaDoc comment, left over from a previous implementation of the compiler in Java. Since we are not using anywhere comments to document the APIs in such detail I decided to remove this one. (I am personally a big believer in commenting all public methods, but this would be a daunting task at this point; moreover, the tooling support in emacs/vi for these comments is very poor.)